BREAKING_CHANGES.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unreleased
 
-> Nothing yet.
+### `malachitebft-engine`
+- Changed the reply channel of `GetValidatorSet` message to `RpcReplyPort<Option<Ctx::ValidatorSet>>`
```

Comment:
```suggestion
- Changed the reply channel of `GetValidatorSet` message to take an `Option<Ctx::ValidatorSet>` instead of `Option<Ctx::ValidatorSet>`.
```